packer/floppy/solaris10/S99startup.sh

```diff
@@ -29,27 +31,26 @@ chmod -R go-rwsx /home/vagrant/.ssh
 echo GSSAPIAuthentication no >> /etc/ssh/sshd_config
 echo LookupClientHostnames no >> /etc/ssh/sshd_config
 
-# install pkgutil because yay
-/usr/sfw/bin/wget --no-check-certificate 'http://get.opencsw.org/now' -O /tmp/pkgutil.pkg
+echo "SSH Installed" >> /tmp/s99log
+
+# install omnibus-build-essential package
+cd /tmp
 echo "mail=\ninstance=overwrite\npartial=nocheck\nrunlevel=nocheck\nidepend=nocheck\nrdepend=nocheck\nspace=nocheck\nsetuid=nocheck\nconflict=nocheck\naction=nocheck\nbasedir=default" > /tmp/noask
```

Comment:
I didn't ask last time, but is it worth doing a `rm -f /tmp/noask` at the end of this script?